test/rules/no-setup-in-describe.js

```diff
@@ -19,10 +19,16 @@ ruleTester.run('no-setup-in-describe', rule, {
         'it("", function () { a[b]; })',
         'it("", function () { a["b"]; })',
         'describe("", function () { it(); })',
+        'describe("", function () { this.slow(1); it(); })',
+        'describe("", function () { this.timeout(1); it(); })',
+        'describe("", function () { this.retries(1); it(); })',
```

Comment:
Can you add one test case with a computed member expression? E.g. `this['retries'](1);`

Reply:
My change to the rule doesn't allow for that form of the expression, so I added this to the invalid test cases, but lmk if you would rather I add support for that form in the rule!

Reply:
IMHO we should handle this case as well. There is already a function for that in ast utils: `getPropertyName()`.

Reply:
All set!